charts/azure-k8s-metrics-adapter/values.yaml

```diff
@@ -46,6 +46,14 @@ azureAuthentication:
   clientCertificate: ""
   clientCertificatePath: ""
   clientCertificatePassword: ""
+  # if you use aadPodIdentity authentication
+  azureIdentityName: "custom-metrics-identity"
+  azureIdentityBindingName: "custom-metrics-identity-binding"
+  # The full Azure resource id of the managed identity (/subscriptions/{SubID}/resourceGroups/{ResourceGroup1}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{IdentityName})
+  azureIdentityResourceId: ""
+  # The Client Id of the managed identity
+  azureIdentityClientId: ""
```

Comment:
I see `azureIdentityName` being used but not `azureIdentityBindingName`, `azureIdentityResourceId`, `azureIdentityClientId`.  Are there files missed in a commit?

Reply:
Good catch. My previous commit missed two new files for identity and identity binding that use these settings values. It was due to an error in the .gitignore file that was ignoring any file in a directory `azure-k8s-metrics-adapter`. It's fixed now.